iterations_runner.lua

```diff
@@ -98,6 +120,14 @@ if #arg ~= 4 and #arg ~= 7 then
     os.exit(1)
 end
 
+-- Add the benchmarks directory as a Lua module search path
+local slashidx = string.find(BM_benchmark, "/[^/]*$")
```

Comment:
OK, so this is where we definitely need some sort of sanity-check assert, as this is definitely fragile (e.g. if Krun hands us `/a/b/c//` this will fail).